### ISSUE TITLE
Illegal byte sequence during composer install, probably due to "hyvä" being used in filename

### ISSUE THREAD

**Issue by hostep** [hyva-themes/magento2-hyva-admin#50]
Hi!

I just noticed when upgrading `hyva-themes/module-magento2-admin` from 1.1.5 to 1.1.14 the following output in the composer installation output:

```
Package operations: 1 install, 0 updates, 0 removals
  - Installing hyva-themes/module-magento2-admin (1.1.14): Extracting archive
    Failed to extract hyva-themes/module-magento2-admin: (50) unzip -qq  'vendor/composer/tmp-bd4f096c2473d75e60620b636d854730' -d 'vendor/composer/d405b1b4'

error:  cannot create vendor/composer/d405b1b4/hyva-themes-magento2-hyva-admin-9262d82/doc/1. Overview/5. Deep Docs/60. Hyv+� Admin Forms (WIP).md
        Illegal byte sequence

    The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
    Unzip with unzip command failed, falling back to ZipArchive class
```

Seems to be that composer can't handle the `Hyvä` filename of [that markdown file](https://github.com/hyva-themes/magento2-hyva-admin/blob/e9142748930ec69ee89e1e99cf848bcd0756c811/doc/1.%20Overview/5.%20Deep%20Docs/60.%20Hyv%C3%A4%20Admin%20Forms%20(WIP).md) on my system.

More environmental info:
- composer 2.0.13
- php 7.4.19
- libzip 1.7.3
- UnZip 6.00
- macOS 10.15.7
- APFS file system that is formatted as case sensitive

It would probably be good to rename that file and change `ä` to `a` to avoid this problem?

**Comment by Vinai** [comment]
Thanks, will do!

**Comment by Vinai** [comment]
Fixed in release 1.1.15

**Comment by hostep** [comment]
Nice, I can confirm that it doesn't happen anymore on version 1.1.15

Thanks for the super fast change!
